### DISCUSSION TITLE
Branding changes

### DISCUSSION THREAD

**Discussion by pepa65** [dani-garcia/vaultwarden#1635] [Q&A]
It would be good to document what changed or will change with the project's branding change to vaultwarden. An important one is changing `/etc/bitwarden_rs.d` to `/etc/vaultwarden.d`. I was wondering about the webvault, is the 'artwork' going to change as well?

**Comment by dani-garcia** [comment]
About the scripts, I forgot about those but plan to keep compatibility with both the old and new name, at least for now. Changed it in 1e5306b8203a7ebe24047910e6c690c18c6d827a

The web vault is going to stay as it is mostly, to keep it the same as the other clients. I've added a (powered by vaultwarden) to the copyright footer of the page to have a way to differentiate it from any official server though.

I'll make a release in a couple of hours and explain the changes there as well.

**Reply by pepa65** [reply]
So the Bitwarden logos stay, and vaultwarden appears only in the footer? Sounds OK to me. (But why do the branding change at all then? Is it OK with upstream to keep using their logo and name branding??)

**Reply by cksapp** [reply]
> Is it OK with upstream to keep using their logo and name branding

Most likely, Kyle the founder of Bitwarden and an excellent coder, is a really good guy. From what I understand he has been very open with forks off of the main project. The code base itself is open source and I think he has always tried to "give away" as much as he can back to the community while still needing to make a buck to eat.

That being said I do think the change is mostly for the sake of keeping the projects separate, while related support for Vaultwarden code should come from Vaultwarden whereas support for Bitwarden applications should be addresses by the Bitwarden team.
People running the unofficial then name Bitwarden_RS may have confusion and end up submitting unrelated support tickets to Bitwarden and not under the issue tracker here in Github for vaultwarden when running Vaultwarden.

**Comment by robartsd** [comment]
I think the web vault branding needs to be changed. I believe that deploying Vaultwarden as currently branded is a violation of [Bitwarden's Trademark Guidelines](https://github.com/bitwarden/server/blob/master/TRADEMARK_GUIDELINES.md). At minimum, the Bitwarden logos (log in page, vault pages, and favicon) should be replaced with Vaultwarden logos. Also the Bitwarden name in the copyright notice should have the registered trademark mark. I'd also like to see some links from the login page to both the official Bitwarden site and the Vaultwarden project.

**Reply by BlackDex** [reply]
I don't know how often we need to tell this, but all is as discussed with Bitwarden. They allow this. The trademark mark isn't there in there own repo, it's not that we removed it or something. We added the powered by Vaultwarden as requested by the Bitwarden team.

You are free to change it for your own project of course.

**Reply by robartsd** [reply]
The Bitwarden developer you talked with is OK with it, but their lawyers should not be okay with public deployments that fail to adhere to the Trademark Guidelines - failure to defend Trademarks can lead to them becoming indefensible. I certainly will change the branding for my own deployments, but this project would be best served removing all instances of the Bitwarden logo from releases and being careful with any use of the Bitwarden name.

**Reply by BlackDex** [reply]
Well that bitwarden developer is the owner of the project. But again. You are free to change anything you deem needed to be changed.
